control-plane/roles/monitoring/templates/grafana_dashboards/lightbits-latency.yaml

```diff
@@ -0,0 +1,267 @@
+---
```

Comment:
Not necessary, lightbits is not part of the metal-stack. All related dashboards can be cleaned up.

Reply:
https://github.com/metal-stack/metal-roles/pull/115/commits/bdb0827a90f520d73aa292b008d6ac00dfa8a182